Gemfile

```diff
@@ -28,7 +28,7 @@ group :development do
   gem 'guard-rake'
   gem 'pry'
   gem 'pry-doc'
-
+  gem 'pry-byebug'
```

Comment:
We have permission from Trevor to add this to static assets since we support Ruby 2.0 now and it's a very helpful tool for debugging. I can remove it if needed, but it will more than likely end up back.